ajax/libs/libphonenumber-js/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "libphonenumber-js",
+  "description": "A simpler (and smaller) rewrite of Google Android's popular libphonenumber library",
+  "version": "0.2.3",
+  "author": {
+    "name": "Halt Hammerzeit",
+    "email": "halt.hammerzeit.at@gmail.com"
+  },
+  "filename": "libphonenumber-js.js",
+  "homepage": "https://github.com/halt-hammerzeit/libphonenumber-js",
```

Comment:
The homepage url is the same as github repository url.
I think that the homepage url `https://halt-hammerzeit.github.io/libphonenumber-js/` is better.